src/harness/BenchmarkDotNet.Extensions/ValuesGenerator.cs

```diff
@@ -84,19 +84,26 @@ private static T GenerateValue<T>(Random random)
         {
```

Comment:
Would it be better to make some of these return their full range (at least the unsigned ones since `Random.Next()` doesn't return negatives)?

Reply:
I was going for consistency with the existing `uint` and `ulong` ones, but I see they were just added last week.  Probably wouldn't be bad to change those now.  One other issue I noticed is that the `double` (and now `float`) one return values between 0 and 1, which is great for most floating point tests, but when you're converting those to integer values, it doesn't give you much coverage.

Reply:
Yeah. I would think ideally we'd have some level of configuration that allows returning `negative values` and values within the entire `floating-point` domain.
The latter would likely still be random, but you wouldn't have "even distribution" (since `floating-point` isn't evenly distributed).